pylxd/image.py

```diff
@@ -80,7 +81,66 @@ def create(cls, client, image_data, public=False, wait=False):
             Operation.wait_for_operation(client, response.json()['operation'])
         return cls(client, fingerprint=fingerprint)
 
+    @classmethod
+    def create_from_config(cls, client, config, wait=False):
+        """ Create an image from the given configuration.
+
+        See: https://github.com/lxc/lxd/blob/master/doc/rest-api.md#post-6
+        """
+        if 'source' not in config or 'fingerprint' not in config['source']:
+            raise KeyError(
+                'You need to provide a fingerprint '
+                'in the config[\'source\'] argument'
+            )
+
+        response = client.api.images.post(json=config)
+        if wait:
+            Operation.wait_for_operation(client, response.json()['operation'])
+        return cls(client, fingerprint=config['source']['fingerprint'])
+
     def export(self):
         """Export the image."""
         response = self.api.export.get()
         return response.content
+
+    def copy_to_lxd(self, new_client, public=None, auto_update=None,
```

Comment:
I would certainly love this functionality (and the same with `Profile`). I think `copy` is a better name for this, but the function definition is probably fine. Is there a reason we can't get `public` and `auto_update` from the properties from the existing image, rather than specifying them here? If not, `public` should default to `False`, rather than `None`.